guides/documentation.md

```diff
@@ -0,0 +1,115 @@
+# Documentation
+
+## Summary
+
+Documentation is central to the development work that we do at the Data Shed.
+Every project should contain enough information to allow for new engineers
+joining the project to understand what is occurring in a given section of the
+code. That said, it does come with the caveat that the documentation should be
+scoped to meet the expectations of the consumer reading the documentation.
+
+### The Audience
+
+The audience is key for all levels of documentation. Links to more detailed
+implementations should be linked to from higher level instruction not the other
+way around. Allow people to get into the details if they choose but do not
+overburden someone with too much information.
+
+## High Level Operational Guides
+
+As a project starts off, we should define high level architectural diagrams of
+how the project is structured. We have used
+[mermaid](https://openbase.com/js/mermaid/documentation) successfully in the
```

Comment:
I think it might be better to link to the "official" Mermaid docs: <https://mermaid-js.github.io>